lib/rollbar.rb

```diff
@@ -363,6 +360,18 @@ def trace_data(exception)
       }
     end
 
+    def exception_backtrace(exception)
+      return exception.backtrace if exception.backtrace.respond_to?( :map )
+      return [] unless configuration.populate_empty_backtraces
+
+      cleaned_backtrace = caller.reject {|path| path =~ /#{rollbar_gem_dir}/ }
```

Comment:
Does this reject all rollbar frames anywhere in the stack? I think we should keep the ones that appear in the middle of the stack (i.e. our middleware).